.github/workflows/docs.yml

```
@@ -0,0 +1,51 @@
+name: AMR-Wind Docs
+
+ on:
+   push:
+     branches:
+       - development
+     paths:
+       - 'amr-wind/**'
+       - 'docs/**'
+       - '.github/workflows/docs.yml'
+   pull_request:
+     branches:
+       - development
+     paths:
+       - 'amr-wind/**'
+       - 'docs/**'
+       - '.github/workflows/docs.yml'
+
+ jobs:
+   Docs:
+     runs-on: ubuntu-latest
+     steps:
+       - uses: actions/checkout@v2
+       - name: Python
+         uses: actions/setup-python@v2
+         with:
+           python-version: '3.x'
+       - name: Dependencies
+         run: |
+           export DEBIAN_FRONTEND=noninteractive
+           sudo apt-get update
+           sudo apt-get install -y --no-install-recommends \
+             doxygen graphviz
+           python3 -m pip install --upgrade pip
+           pip3 install sphinx sphinx_rtd_theme
+       - name: Build
+         working-directory: ${{github.workspace}}
+         run: |
+           sphinx-build -M html ./docs/sphinx ./build_docs/manual
+           doxygen ./docs/doxygen/Doxyfile
+           mv ./build_docs/manual/html ./documentation
+           mv ./build_docs/doxygen/html ./documentation/api_docs
+           touch ./documentation/.nojekyll
+       - name: Deploy
+         if: github.event_name == 'push' && github.ref == 'refs/heads/development'
```

```suggestion
         if: github.event_name == 'push'
```

Can be simplified as the workflow won't run on any other branches anyway due to `on:` rules.